tests/wpt/mozilla/tests/mozilla/set_zoom_factor_from_test_ref.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
```

Comment:
The `<html>` and `</html>` tags are unneeded, here and in the test, feel free to remove them